components/common.css

```diff
@@ -0,0 +1,7 @@
+.item-conainter{
```

Comment:
container spelling, do give some padding to main-section class

Reply:
@nisargpawade Container spelling corrected
Padding added for main section